src/components/Button.tsx

```diff
@@ -245,7 +234,7 @@ export default styled(Button)`
   display: flex;
   align-items: center;
   justify-content: center;
-  height: ${({ squared, short }) => (squared || short ? '40px' : 'var(--inputHeight)')};
+  height: ${({ squared, short }) => (short ? '40px' : squared ? '40px' : 'var(--inputHeight)')};
```

Comment:
no need to change this but I am curious why you didn't like the previous implementation 😅

Reply:
Haha, I was '35px' for short, up until 1h ago... Just trying things & going back and forth to see how it looks.